src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/PatientEverythingService.cs

```diff
@@ -177,6 +223,45 @@ public class PatientEverythingService : IPatientEverythingService
             return new SearchResult(searchResultEntries, searchResult.ContinuationToken, searchResult.SortOrder, searchResult.UnsupportedSearchParameters);
         }
 
+        private void StoreSeeAlsoLinkInformationInToken(string resourceId, EverythingOperationContinuationToken token, IReadOnlyCollection<SearchResultEntry> searchResultEntries)
+        {
+            SearchResultEntry patientResource = searchResultEntries.FirstOrDefault(s => string.Equals(s.Resource.ResourceTypeName, ResourceType.Patient.ToString(), StringComparison.Ordinal));
+
+            if (patientResource.Resource != null)
+            {
+                ResourceElement element = _resourceDeserializer.Deserialize(patientResource.Resource);
+                Patient patient = element.ToPoco<Patient>();
+
+                List<Patient.LinkComponent> links = patient.Link;
+
+                if (links != null)
+                {
+                    foreach (Patient.LinkComponent link in links)
```

Comment:
What is the expected behavior if the patient has multiple links (like having a "seealso" link and a "replaceby" link at the same time)?

Reply:
Ah, thanks for bringing this up. I made the assumption that an error should be returned if one or more `replaced-by` links are present regardless of the other links. I will confirm with @CaitlinV39!

Reply:
Agree with Robin!

Reply:
Thanks! If so I think we can add some comments in relative codes to describe this behavior.

Reply:
Please let me know how the comments look in a8f5b52.